config/application.rb

```diff
@@ -44,5 +44,7 @@ class Application < Rails::Application
 
     # This makes sure our Postgres enums function are persisted to the schema
     config.active_record.schema_format = :sql
+
+    config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT)) if ENV['RAILS_LOG_TO_STDOUT'].present?
```

Comment:
If I'm reading this right, STDOUT will be used to log whether the `RAILS_LOG_TO_STDOUT` environment variable is set to true or false (because all keys and values in the `ENV` hash are string, AFAIK). I wonder if `ENV['RAILS_LOG_TO_STDOUT'].present?` should be something like `ENV['RAILS_LOG_TO_STDOUT'] == 'true'`?